### PR TITLE
direnv: Use the same nix package

### PR DESCRIPTION
**Pull request by srid**
In direnv, use the same Nix package as the user specifies here:

https://github.com/juspay/nix-dev-home/blob/d9df14e49eb7b291584dfb0a5df1323754bd7644/home/default.nix#L9-L12

context:

image

### CODE REVIEW

File: home/default.nix [hunk 1]
```diff
@@ -121,6 +121,7 @@
     # https://nixos.asia/en/direnv
     direnv = {
       enable = true;
+      package = config.nix.package;
```
Comment (shivaraj-bh) [inline home/default.nix:124]: Isn't it nix-direnv.package instead of direnv.package

Comment (srid) [reply home/default.nix:124]: Ah, yes, this PR broke stuff. 

But your change is also not correct?

image

image

Comment (srid) [reply home/default.nix:124]: It is already using `config.nix.package`. I'll revert this PR for now.

Comment (srid) [reply home/default.nix:124]: Actually, this was my fault for not paying attention to Rana's actual change:

image

Comment (shivaraj-bh) [reply home/default.nix:124]: @srid the nix-direnv configuration above is from nixos modules and not from home-manager. 

This has to be fixed in home-manager to match nixos module.

**Pull request merged**